### PR TITLE
parse SA for tf runner, in gcp case

### PR DESCRIPTION
**Pull request by eschultink**
### Features
 - parse 'gcp_terraform_sa_account_email' from `terraform.tfvars`, to support GCP terraform applies via SA impersonation

### Change implications

 - dependencies added/changed? **no**
 - something to note in `CHANGELOG.md`? **no**

### CODE REVIEW

File: infra/modules/gcp-tf-runner/main.tf [hunk 1]
```diff
@@ -30,6 +30,11 @@ data "google_service_account_id_token" "identity" {
   target_audience = "worklytics.co/gcp-tf-runner"
 }
 
+# as this runs `terraform console` underneath, it is hacky and SLOW!!!
+data "external" "identity" {
```
Comment (jlorper) [inline infra/modules/gcp-tf-runner/main.tf:34]: thinking something like this should exist just found this -> https://registry.terraform.io/providers/adminsentecacommerce/tfvars/latest

Not good examples on how to use though

File: infra/modules/gcp-tf-runner/main.tf [hunk 2]
```diff
@@ -30,6 +30,11 @@ data "google_service_account_id_token" "identity" {
   target_audience = "worklytics.co/gcp-tf-runner"
 }
 
+# as this runs `terraform console` underneath, it is hacky and SLOW!!!
```
Comment (jlorper) [inline infra/modules/gcp-tf-runner/main.tf:33]: seems you don't use `terraform console` anymore, right? remove this line?

**Review by jlorper — APPROVED** [review]

File: infra/modules/gcp-tf-runner/main.tf (hunk 1 above)
Comment (jlorper) [reply infra/modules/gcp-tf-runner/main.tf:34]: Also this -> https://terraform-docs.io/reference/tfvars-json/

Comment (eschultink) [reply infra/modules/gcp-tf-runner/main.tf:33]: That would be fine for internal-use, but we need to minimize dependencies, esp those published by small 3rd parties.

**Pull request merged**